src/components/MangaBar/manga-bar.css

```diff
@@ -0,0 +1,41 @@
+.round-start {
+  border-start-start-radius: 0.25rem;
+  border-end-start-radius: 0.25rem;
+}
+
+.round-end {
+  border-start-end-radius: 0.25rem;
+  border-end-end-radius: 0.25rem;
+}
+
+.covers {
+  padding: 0.5rem;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 2.5rem);
+  grid-template-rows: 2.5rem;
+  grid-auto-rows: 0;
+  gap: 0 0.5rem;
+}
+
+.hover-covers {
```

Comment:
Merge common part of `covers` and `hover-covers` into `covers` and create `hover` and `shown` classes for the variants